src/app/component/user/components/profile/calendar/calendar.component.scss

```diff
@@ -233,20 +168,12 @@ $bp-largest: 1200px;
 
 .unenrolled-past-day {
   background-color: transparent;
-  border: 1px solid  #d0d0d0;
+  border: 1px solid #d0d0d0;
```

Comment:
move color to theme folder/file

Reply:
Thanks. When I fixed it, I found more colors.
Fixed.